src/client/components/Delete_Comfim/Button.component.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Button from '../button/button.component';
+import './Style.css';
+import ModalList from './ModalList.component';
+import Modal from './Modal';
+
+const ButtonComponent = () => {
+  const { isShowing, toggle } = Modal();
+  return (
+    <div>
+      <Button className="base-btn" type="danger" onClick={toggle}>
+        Delete BATCH
+      </Button>
+      <ModalList isShowing={isShowing} hide={toggle} onClose={toggle} />
```

Comment:
Please see the comments I left on the `ModalList` component. 
If you applied those changes, this could just be
```
{isShowing ? <ModalList setIsShowing={setIsShowing} onDelete={() => null}/> : null}
```